pipeline_dp/aggregate_params.py

```diff
@@ -61,7 +61,6 @@ class AggregateParams:
     max_contributions_per_partition: int
     low: float = None
     high: float = None
-    budget_weight: float = 1
```

Comment:
I think it makes sense to keep it in params as well (but to fix it :) ). If a user have some simple usecase - 2 aggregations, the user it would be easier just to specify this parameter, instead of learning how to work with scopes . WDYT?

Reply:
This makes sense. Implemented this by creating a scope internally.